metadata-service/factories/src/main/resources/application.yml

```diff
@@ -36,9 +39,13 @@ authorization:
 
   # Optional: A set of custom authorizers, serving in addition to the default DataHub policies-based authorizer.
   authorizers:
```

Comment:
It would be more ideal if we built a way to overwrite the applcation.yml.. 

Maybe we can address this in a followup?

Reply:
We can address it in Pluggable Authorizer functionality.